src/Ssh/Client.php

```diff
@@ -180,7 +180,7 @@ private function createProcess($command)
         if (method_exists('Symfony\Component\Process\Process', 'fromShellCommandline')) {
             return Process::fromShellCommandline($command);
         } else {
-            return new Process($command);
+            return new Process(explode(' ', $command));
```

Comment:
I’m not sure this is proper way. There is a fromSgellCommand method.

Reply:
Are you sure there is only one compatibility issue? Could it be better to change the version for Process in the composer?

Reply:
You are right, but if you do that, you don´t need else bloc statement